### ISSUE TITLE
After call endActivity can't create new one until restart the app

### ISSUE THREAD

**Issue by adham-ashraf77** [istornz/flutter_live_activities#106]
I have a case in which I need a live activity for every current order so when the order is finished and the app is still running and comes with a new one I call to create new live activity but it does not appear to have any help with this issue.

**Comment by adham-ashraf77** [comment]
@istornz any help please to have to show a new live activity with the same activity id where ended?

**Comment by istornz** [comment]
Hello, thanks for raising an issue, do you have some errors? Is the live activity is correctly dissmissed when you call `endActivity()`?

**Comment by adham-ashraf77** [comment]
> Hello, thanks for raising an issue, do you have some errors? Is the live activity is correctly dissmissed when you call `endActivity()`?

yes it dismissed when call endActivity() but when I call create or update to show new live activity not show until I restart the app

**Comment by dimitar-zabaznoski** [comment]
This probably happens because you're using the same ID to restart the live activity once removed. Why not just use a new ID after calling endActivity? Does this solve your problem?
